### PR TITLE
[Student][Interaction tests][MBL-13039] | DashboardInteractionTest

### PR DESCRIPTION
**Pull request by joehoag**

### CODE REVIEW

File: apps/student/src/main/res/layout/viewholder_announcement_card.xml [hunk 1]
```diff
@@ -51,8 +51,8 @@
 
         <FrameLayout
             android:id="@+id/buttonContainer"
-            android:layout_width="24dp"
-            android:layout_height="24dp"
+            android:layout_width="48dp"
+            android:layout_height="48dp"
```
Comment (joehoag) [inline apps/student/src/main/res/layout/viewholder_announcement_card.xml]: This was an accessibility bug.

File: apps/student/src/main/java/com/instructure/student/fragment/AllCoursesFragment.kt [hunk 2]
```diff
@@ -48,7 +48,7 @@ import kotlinx.android.synthetic.main.fragment_all_courses.*
 import kotlinx.android.synthetic.main.panda_recycler_refresh_layout.*
 import org.greenrobot.eventbus.EventBus
 import org.greenrobot.eventbus.Subscribe
-import kotlinx.android.synthetic.main.fragment_all_courses.fragment_container as rootView
+import kotlinx.android.synthetic.main.fragment_all_courses.all_courses_fragment_container as rootView
```
Comment (StephenBrough) [inline apps/student/src/main/java/com/instructure/student/fragment/AllCoursesFragment.kt]: Could we make this camel case?

File: apps/student/src/main/res/layout/fragment_all_courses.xml [hunk 3]
```diff
@@ -18,7 +18,7 @@
 <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:id="@+id/fragment_container"
+    android:id="@+id/all_courses_fragment_container"
```
Comment (StephenBrough) [inline apps/student/src/main/res/layout/fragment_all_courses.xml]: I guess here's where the change should be made 😄

Comment (joehoag) [reply apps/student/src/main/res/layout/fragment_all_courses.xml]: :+1: Fixed in latest commit.

**Pull request merged**